pallets/automation-time/src/lib.rs

```diff
@@ -469,17 +469,6 @@ pub mod pallet {
 				T::MaxWeightPercentage::get().mul_floor(T::MaxBlockWeight::get());
 			Self::trigger_tasks(max_weight)
 		}
-
-		fn on_runtime_upgrade() -> Weight {
-			let on_chain_storage_version = StorageVersion::get::<Pallet<T>>();
-			info!("on chain storage version, {:?}", on_chain_storage_version);
```

Comment:
we don't need on chain storage version?

Reply:
We will be doing a different pattern for upgrades.  I will add in needed code to the double map PR